### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by rainsworth** [softwaresaved/lowfat#621]
The following were issues/suggestions sent in (not sure if they will be addressed during the rest of the accessibility fixes):
- [x] In the nav bar, the ‘+’ icon for the drop-down list isn't screen reader/voice control friendly. It looks like there's no aria-label on it, so I can’t click it by voice. Also, from a voice control perspective, it's hard to guess what aria-label a ‘+’ icon will have in general. Is it ‘add’ or ‘new’ or ‘create’ or… every website is different! The ideal for me would be for it to be a text label instead, or an icon plus text.
- [x] The same applies to the icons for websites/orcid/twitter etc. on the main page (these are harder to replace with visible text but should have some aria-label like ‘Twitter for <fellow>’ etc.)
- [x] On another note, the Accessibility Statement linked at the bottom of the page doesn't seem to be publicly accessible.

**Comment by jag1g13** [comment]
Point two has been addressed in the recent pull request to our main sprint branch as this was picked up by Little Forest and was the cause of many hundreds of errors. I'll have a look at the '+' icon on Monday

**Comment by phillybroadbent** [comment]
Added text label 'Options' to the '+' icon so that the drop down list is more accessible for people using screen readers.
